src/main/java/de/medizininformatikinitiative/torch/model/ccdl/AttributeFilter.java

```diff
@@ -0,0 +1,30 @@
+package de.medizininformatikinitiative.torch.model.ccdl;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+import java.util.List;
+import java.util.Objects;
+
+
+@JsonInclude(Include.NON_NULL)
+@Builder
+public record AttributeFilter(
```

Comment:
Please do not copy all the CCDL classes here. A simple JsonNode will do it.

Reply:
@alexanderkiel  are you sure - the sq2cql translator needs a StructuredQuery as input